hedera-mirror-importer/src/main/resources/db/scripts/init.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -e
+
+PGCONF="${PGCONF:-/var/lib/postgresql/data}"
+PGHBA="${PGCONF}/pg_hba.conf"
+USER_SQL="alter user :ownerUsername with createrole;"
```

Comment:
nit: maybe a more descriptive variable Name here
```suggestion
USER_CREATE_OR_ALTER_SQL="alter user :ownerUsername with createrole;"
```

Reply:
Didn't like that name either. Changed it to `DB_SPECIFIC_SQL` to leave it open for future db specific statements added to it.